### ISSUE TITLE
Feature Idea: Color-code stats page wins/losses

### ISSUE THREAD

**Issue by shaunkatona** [mtgred/netrunner#5873]
It's hard to quickly see what games I won and what games I lost looking at my stats games/history.  I wrote a bookmarklet that helps me, but maybe others would benefit from something like this:

![image](https://user-images.githubusercontent.com/6580631/117527287-0e952300-af88-11eb-973f-44f285ec95e4.png)

**Comment by lallmon** [comment]
Please share your bookmarklet code!

**Comment by shaunkatona** [comment]
```
javascript: (function a() {
    const avatar = document.querySelector('#right-menu > ul > li > a > img');
    const username = avatar.getAttribute('alt');
    const games = document.querySelectorAll('div.gameline');
  
    games.forEach(game => {
        const players = game.querySelectorAll('span.player img.avatar');
        const corpName = players[0].getAttribute('alt');
        const runnerName = players[1].getAttribute('alt');
        const winner = game.querySelector('h4:nth-child(6)');
      
        if (winner && runnerName) {
            if (winner.innerText.toLowerCase().includes('runner')) {
                if (runnerName === username) {
                    game.style.backgroundColor = '#618b61';
                    return;
                }
            }
            game.style.backgroundColor = '#faa2a2';
        }
    });
})();
```

**Comment by NBKelly** [comment]
Closed as completed via #7406
